modules/dxp/apps/scim/scim-rest-impl/src/main/java/com/liferay/scim/rest/internal/manager/UserManagerImpl.java

```diff
@@ -787,8 +784,14 @@ private ScimUser _getScimUser(long companyId, long userId)
 	private UserGroup _getUserGroup(long companyId, long userGroupId)
 		throws AbstractCharonException {
 
-		UserGroup userGroup = _userGroupLocalService.fetchUserGroup(
-			userGroupId);
+		UserGroup userGroup = null;
+
+		try {
+			userGroup = _userGroupService.fetchUserGroup(userGroupId);
+		}
+		catch (PortalException portalException) {
+			throw new ForbiddenException(portalException.getMessage());
```

Comment:
I'm not sure if we want to do it like this but as only the permission check can throw this exception I used a `ForbiddenException` but we can change it if it reveals too much information.